<launch>
  <param name="/use_sim_time" value="true"/>

  <!-- Remap if your topics differ -->
  <node pkg="edge_detection" type="edge_ros_node.py" name="edge_node" output="screen">
    <param name="rgb_topic"  value="/camera/color/image_raw"/>
    <param name="depth_topic" value="/camera/depth/image_rect_raw"/>
    <param name="info_topic" value="/camera/color/camera_info"/>
  </node>

  <node pkg="edge_detection" type="edge_dir_service.py" name="edge_dir_service" output="screen"/>
</launch>

